test/testvm.py

```diff
@@ -709,7 +722,7 @@ def _read_network_name(self):
     # only read the disk template once per machine
     def _domain_disk_template(self):
         if not self.test_disk_desc_original:
-            with open("./guest/test-domain-disk.xml", "r") as desc_file:
+            with open(os.path.join(self.test_dir, "/guest/test-domain-disk.xml"), "r") as desc_file:
```

Comment:
> Path needs to be relative.

Meh, copy/paste error. I did that in another place and running the test found it.